### PR TITLE
filebeat/module/cef: import genericised cef dashboards from integrations

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

This adds the dashboards that exist in the CEF integration to the CEF module in filebeat.

## Why is it important?

The existing dashboards do not work in the general case as they were originally written against the ArcSight fields that the original module worked with.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ] Query how to deal with the existing kibana/7 dashboards.

## Related issues

- Closes elastic/integrations#3469
- Relates elastic/integrations#3526

## Screenshots

See screenshots in elastic/integrations#3526.

### CODE REVIEW

**Comment by efd6** [comment]
https://github.com/jitterbit/get-changed-files/issues/4

**Review by andrewkroh — CHANGES_REQUESTED** [review]
I think the dashboards require some changes to work with Beats.

- Beats uses `event.dataset` instead of `data_stream.dataset`.
- The index pattern is `filebeat-*` rather than `logs-*`.
- Are the object IDs the same as what's in the Fleet integration? We should make them unique to prevent collisions Kibana with Fleets saved objects.

**Comment by efd6** [comment]
I have deconflicted the object IDs with clonedash (second commit) and the index patterns have been adjusted (last commit). I'll look into the `data_stream.dataset` => `event.dataset`.

File: x-pack/filebeat/module/cef/_meta/kibana/8/dashboard/cef-1ef1282f-382b-4bdd-9cb1-810b8f083e74.json [hunk 1]
```diff
@@ -301,11 +301,191 @@
...
-                        "mapStateJSON": "{\"zoom\":1.78,\"center\":{\"lon\":0,\"lat\":16.40767},\"timeFilters\":{\"from\":\"now-24h\",\"to\":\"now\"},\"refreshConfig\":{\"isPaused\":true,\"interval\":0},\"query\":{\"query\":\"\",\"language\":\"kuery\"},\"filters\":[],\"settings\":{\"autoFitToDataBounds\":false,\"backgroundColor\":\"#ffffff\",\"disableInteractive\":false,\"disableTooltipControl\":false,\"hideToolbarOverlay\":false,\"hideLayerControl\":false,\"hideViewControl\":false,\"initialLocation\":\"LAST_SAVED_LOCATION\",\"fixedLocation\":{\"lat\":0,\"lon\":0,\"zoom\":2},\"browserLocation\":{\"zoom\":2},\"maxZoom\":24,\"minZoom\":0,\"showScaleControl\":false,\"showSpatialFilters\":true,\"showTimesliderToggleButton\":true,\"spatialFiltersAlpa\":0.3,\"spatialFiltersFillColor\":\"#DA8B45\",\"spatialFiltersLineColor\":\"#DA8B45\"}}",
+                        "layerListJSON": [
```
Comment (efd6) [inline x-pack/filebeat/module/cef/_meta/kibana/8/dashboard/cef-1ef1282f-382b-4bdd-9cb1-810b8f083e74.json:304]: This looks to me like it should not happen. Others below.

Comment (andrewkroh) [reply x-pack/filebeat/module/cef/_meta/kibana/8/dashboard/cef-1ef1282f-382b-4bdd-9cb1-810b8f083e74.json:304]: This is something unique to Beats. The stringified JSON is expanded such that it's easier to diff in git. When the dashboards are loaded by the `<beat> setup` command it turns it back into a string.

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**